Sources/Mapper.swift

```diff
@@ -165,7 +165,7 @@ public final class Mapper<N: BaseMappable> {
 		#if swift(>=4.1)
 		let result = JSONArray.compactMap(map)
 		#else
-		let result = JSONArray.flatMap(map)
+		let result = JSONArray.compactMap(map)
```

Comment:
**Same comment as above.**
Please just remove `else` statement if it is necessary because now `if` and `else` blocks has same actions.